docs/other-topics/dialect-specific-things.md

```diff
@@ -262,6 +262,36 @@ const sequelize = new Sequelize('MY_DSN', 'username', 'password', {
 
 The final connection string generated by the above configuration would look like `CMT=1;NAMING=0;...;DSN=MY_DSN;UID=username;PWD=password;`. Additionally, the `host` option will map the the `SYSTEM=` connection string key.
 
+### Oracle Database
+
+The underlying connector library used by Sequelize for Oracle is the [node-oracledb](https://www.npmjs.com/package/oracledb) package.  
+See [Releases](/releases#oracle-support-table) to see which versions of Oracle Database & node-oracledb are supported.
+
+node-oracledb needs [instant client library files](https://www.oracle.com/database/technologies/instant-client/downloads.html) to work. You can use node-oracledb [quick start](https://oracle.github.io/node-oracledb/INSTALL.html#quickstart) link for installations.
+
+Below is Sequelize constructor with parameters related to Oracle Database.
+
+```js
+const sequelize = new Sequelize('servicename', 'username', 'password', {
+  dialect: 'oracle',
+  host: 'hostname',
+  [port: 'port number',]
+});
+```
+
+The default port number for Oracle database is 1521.
+
+Sequelize also lets you pass credentials in URL format:
+
+```js
+const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
+```
+
+:::note
+
+The `Sequelize.Datatypes.TEXT` doesn't support the `where` option. 
```

Comment:
```suggestion
`DataTypes.TEXT` doesn't support the `where` option. 
```

I think this belongs on the other-data-types page instead

Reply:
A note is added in other-data-types